examples/discrete_turbines/tidal_array.py

```diff
@@ -0,0 +1,108 @@
+from thetis import *
```

Comment:
Could you add some comments at the start that describe what setup is defined here?

Reply:
Done